pages/cloud/using.md

```diff
@@ -2,25 +2,19 @@
 
 ### Managing your Project with the Cloud Dashboard
 
-Once you've requested and been granted an account on the LakeEffect cloud, you will be able to login in to the web console.
-Your username and password for LakeEffect are the same as your CCR username and password.  However, you need additional 
-privileges to be allowed to login to the cloud.  You will be informed when your cloud account is ready.
+Allocations to the LakeEffect Cloud are requested and managed through the [ColdFront Resource Allocations Management Tool](../portals/coldfront.md).  
```

Comment:
Can you link to getting-access.md#allocation-requests

So this should be: 

To request an allocation to LakeEffect [see here](getting-access.md#allocation-requests)

Reply:
done